internal/ratelimit/throttle_reader_test.go

```diff
@@ -306,22 +305,22 @@ func (t *ThrottledReaderTest) WrappedReturnsShortRead_SecondSucceedsInFull() {
 	// Call
 	n, err := t.reader.Read(make([]byte, 16))
 
-	ExpectEq(16, n)
-	ExpectEq(nil, err)
+	assert.Equal(t.T(), 16, n)
+	assert.NoError(t.T(), err)
 }
 
-func (t *ThrottledReaderTest) ReadSizeIsAboveThrottleCapacity() {
+func (t *ThrottledReaderTest) TestReadSizeIsAboveThrottleCapacity() {
 	buf := make([]byte, 2048)
-	AssertGt(len(buf), t.throttle.Capacity())
+	assert.Greater(t.T(), uint64(len(buf)), t.throttle.Capacity())
 
 	// Wrapped
 	var readCalled bool
 	t.wrapped.f = func(p []byte) (n int, err error) {
-		AssertFalse(readCalled)
+		assert.False(t.T(), readCalled)
 		readCalled = true
 
-		AssertEq(&buf[0], &p[0])
-		ExpectEq(t.throttle.Capacity(), len(p))
+		assert.Equal(t.T(), &buf[0], &p[0])
+		assert.Equal(t.T(), t.throttle.Capacity(), uint64(len(p)))
```

Comment:
Nit: https://pkg.go.dev/github.com/stretchr/testify/assert#Equal

Reply:
If it's about order then the order that is correct t.throttle.Capacity is expected and len(p) is actual.

Reply:
I felt vice versa.